CHANGELOG.md

```diff
@@ -1,5 +1,101 @@
 # Change Log
 
+## [0.9.0](https://github.com/confluentinc/ksql/releases/tag/v0.9.0-ksqldb) (2020-05-08)
```

Comment:
nit: don't forget to update the date here to the actual release date, if that turns out to not be today.

Reply:
done